### PR TITLE
Update getting started guide to work on any Kubernetes cluster

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR updates the getting started guide of Scalar Helm Chart. The summary of updates is the following.

* Can work on any Kubernetes cluster
    * In the previous document, we need to use `minikube + docker` environment. However, these new documents can work on any Kubernetes cluster since all components (Scalar products, backend DB, and client) will be deployed on the Kubernetes cluster.
* Use Marketplace instead of GitHub Packages to pull the container images.
    * I added the description to get container images from AWS Marketplace or Azure Marketplace and the configuration file for each marketplace.
* Use `eclipse-temurin:8` as a client instead of `ubuntu:20.04`
    * If we use `ubuntu:20.04` as a client container, we need to install `openjdk-8-jdk`, but it takes a long time. Now, we use `eclipse-temurin:8` that includes JDK from the beginning. It can save time and make steps easier.
* Add description to access each web console if we use other Kubernetes cluster than minikube
    * If we use other Kubernetes cluster than minikube, we cannot use the `minikube tunnel` command. So, I added the description of the examples to expose ports (using a Load Balancer provided by cloud service or the `kubectl port-forward` command).
* Update sample commands based on the latest getting started of ScalarDB
    * I updated the steps to run the sample application of ScalarDB based on the following PR.
    * https://github.com/scalar-labs/scalardb/pull/617
* Update product name
    * Scalar DB -> ScalarDB
    * Scalar DL -> ScalarDL
* Add how to deploy Ledger in the Auditor documents
    * In the previous document, we need to refer to the two documents alternately (Ledger's document and Auditor's document) if we want to deploy ScalarDL Auditor. However, it was a bit confusing since there are some different things in the configuration of Ledger (scalar.dl.ledger.auditor.enabled=true) and operation to run sample contracts (we need to register certs of Ledger and Auditor) between Ledger only deployment and Auditor mode deployment. To make this document easier, I added the description how to deploy Ledger (with configuration for Auditor) in the Auditor's document. There are some redundant things, but I think it is easier for users since the operation is completed in a single document.
* Update product version
    * I updated the version in each step based on ScalarDB v3.7.0 and ScalarDL v3.6.0.
    * Also, I updated the command outputs of sample contracts based on the above version.

Please take a look!

### CODE REVIEW

**Comment by kota2and3kan** [comment]
@supl  
I also updated the getting started guide of monitoring, logging, and Scalar Manager.
Please take a look!

File: docs/getting-started-monitoring.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 # Getting Started with Helm Charts (Monitoring using Prometheus Operator)
 
-This document explains how to get started with Scalar products monitoring on Kubernetes using Prometheus Operator (kube-prometheus-stack). Here, we assume that you already have a Mac or Linux environment for testing.
+This document explains how to get started with Scalar products monitoring on Kubernetes using Prometheus Operator (kube-prometheus-stack). Here, we assume that you already have a Mac or Linux environment for testing. And, we assume that you can use any Kubernetes cluster, but we use **Minikube** in this document.
```
Comment (feeblefakie) [inline docs/getting-started-monitoring.md:3]:
```suggestion
This document explains how to get started with Scalar products monitoring on Kubernetes using Prometheus Operator (kube-prometheus-stack). Here, we assume that you already have a Mac or Linux environment for testing. We use **Minikube** in this document, but the steps we will show should work in any Kubernetes cluster.
```

**Review by feeblefakie — COMMENTED** [review]
Thank you!
I haven't gone through all the docs, but we can revise it little by little as needed.
Left some minor suggestions.
Please apply the suggestions to all the other related documents that have the same descriptions.

**Comment by kota2and3kan** [comment]
@feeblefakie  
Thank you for your review!
I applied the suggestions to all documents.
Please take a look again!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Comment by feeblefakie** [comment]
@supl I'm merging it but please check when yo get a chance!

**Pull request merged**